### ISSUE TITLE
Installing packages for tooling via npm. An unhandled exception occurred: Package install failed. See "/tmp/ng-kx98Hb/angular-errors.log" for further details.

### ISSUE THREAD

**Issue by sllegendre** [angular/angular-cli#20559]
# 🐞 Bug report

### Command (mark with an `x`)

ng update @angular/core @angular/cli --verbose

- [ ] new
- [ ] build
- [ ] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [x] update
- [ ] lint
- [ ] extract-i18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc

### Is this a regression?

n/a or unkown

### Description

I cannot migrate from 10.1.7 to latest (11.x)

## 🔬 Minimal Reproduction
run: 
ng update @angular/core @angular/cli --verbose 

(same behaviour as sudo or without the --verbose) 

## 🔥 Exception or Error
<pre><code>
Locating potential npmrc files:
Trying '/snap/node/4150/etc/npmrc'...found.
Trying '/root/.npmrc'...not found.
Trying '/home/.npmrc'...not found.
Trying '/home/username/.npmrc'...not found.
Trying '/home/username/Documents/.npmrc'...not found.
Trying '/home/username/Documents/code/.npmrc'...not found.
Trying '/home/username/Documents/code/projectfolder/.npmrc'...not found.
The installed local Angular CLI version is older than the latest stable version.
Installing a temporary version to perform the update.
Installing packages for tooling via npm.
An unhandled exception occurred: Package install failed.
See "/tmp/ng-kx98Hb/angular-errors.log" for further details.

</code></pre>

## 🌍 Your Environment
<pre><code>
Angular CLI: 10.1.7
Node: 14.16.1
OS: linux x64

Angular: 10.1.6
... animations, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router
Ivy Workspace: Yes

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1001.7
@angular-devkit/build-angular   0.1001.7
@angular-devkit/core            10.1.7
@angular-devkit/schematics      10.1.7
@angular/cdk                    10.2.7
@angular/cli                    10.1.7
@angular/material               10.2.7
@schematics/angular             10.1.7
@schematics/update              0.1001.7
rxjs                            6.6.3
typescript                      4.0.5

</code></pre>

**Anything else relevant?**
My machine runs on Linux mint 20.1 cinnamon
I had a look at the logs but I do not find the place where it tells me which exact node package is the culprit. 

Content of log:
<pre><code>
[error] Error: Package install failed.
    at installPackage (/home/sllegendre/Documents/code/projectfoldername/node_modules/@angular/cli/tasks/install-package.js:40:15)
    at installTempPackage (/home/sllegendre/Documents/code/projectfoldername/node_modules/@angular/cli/tasks/install-package.js:78:5)
    at Object.runTempPackageBin (/home/sllegendre/Documents/code/projectfoldername/node_modules/@angular/cli/tasks/install-package.js:83:33)
    at UpdateCommand.run (/home/sllegendre/Documents/code/projectfoldername/node_modules/@angular/cli/commands/update-impl.js:210:38)
    at async UpdateCommand.validateAndRun (/home/sllegendre/Documents/code/projectfoldername/node_modules/@angular/cli/models/command.js:134:28)
    at async Object.runCommand (/home/sllegendre/Documents/code/projectfoldername/node_modules/@angular/cli/models/command-runner.js:198:24)
    at async default_1 (/home/sllegendre/Documents/code/projectfoldername/node_modules/@angular/cli/lib/cli/index.js:59:31)
</pre></code>

**Comment by alan-agius4** [comment]
Can you please provide the contents of `angular-errors.log`?

**Comment by sllegendre** [comment]
After playing around for quite a while, I think I found the area where the root cause lies: My node version was installed through snapcraft. 
This apparently introduced inconsistencies that led to this issue. I was not able or willing to spend more time to resolve it or understand it thoroughly but in the interest of time removed the snap of node and installed node again "normally" through apt-get. 
Closing issue, as it is likely snapcraft related not an angular issue.
